packages/react-dev-overlay/src/middleware.ts

```diff
@@ -11,7 +11,7 @@ import { StackFrame } from 'stacktrace-parser'
 import url from 'url'
 // @ts-ignore
 // eslint-disable-next-line import/no-extraneous-dependencies
-import webpack from 'webpack'
+import webpack from 'webpack4'
```

Comment:
We can now incrementally move all these webpack4 imports to webpack5. Step by step.